client/src/components/AlertSnackbar/AlertSnackbar.tsx

```diff
@@ -12,14 +13,39 @@ const AlertSnackbar = () => {
   const { open, setOpen, alertMessage, alertSeverity } = useContext(
     AlertContext
   );
+  const [duration, setDuration] = React.useState<number>(0);
   const anchorOrigin: SnackbarOrigin = {
     vertical: "top",
     horizontal: "center",
   };
 
+  const handleClose = (event?: React.SyntheticEvent, reason?: string) => {
+    if (reason === "clickaway") {
+      return;
+    }
+    setOpen(false);
+  };
+
+  React.useEffect(() => {
+    if (!open) {
+      setDuration(0);
+    }
+  }, [open]);
+
+  React.useEffect(() => {
+    if (open && alertMessage) {
```

Comment:
What would happen if 
open is true, while there is no alertMessage? Is this expected?

Reply:
The alert would just show the severity icon with no message, which I'm presuming we would not ever intend, so a "new" alert is only when the alertMessage is non-empty